python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md

```diff
@@ -0,0 +1,133 @@
+---
+author: stefkn
+
+levels:
+
+  - beginner
+
+  - basic
+
+  - medium
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+  - workout
+  - deep
+
+links:
+
+  - '[Python sorted](https://www.programiz.com/python-programming/methods/built-in/sorted){website}'
+  - '[Sorting Mini-HOW TO](https://wiki.python.org/moin/HowTo/Sorting/){website}'
+  - '[Timsort](https://en.wikipedia.org/wiki/Timsort){website}'
+  - '[PEP 3000](https://www.python.org/dev/peps/pep-3100/#id100){website}'
+  - '[__cmp__ in Python 2 and 3](https://stackoverflow.com/questions/8276983/python-2-and-python-3-cmp){website}'
+
+
+---
+
+# The `sorted` Built-in Function (Part 2)
+
+---
+## Content
+
+`sorted` can also be used on strings, where Python lexicographically orders individual characters by comparing the Unicode code point number.
+
+```python
+pyString = ',.()-.12345ABCDabcd='
+print(sorted(pyString))
+# result:
+# ['(', ')', ',', '-', '.', '.',
+#  '1', '2', '3', '4', '5', '=',
+#  'A', 'B', 'C', 'D',
+#  'a', 'b', 'c', 'd']
+```

```

Comment:
this should've been in part 1. at the moment part 2 has ~320 words, so moving this section to part 1 will reduce that to a more acceptable value

Reply:
should be inserted before this paragraph:
> Behind the scenes, Python calls a derivative of merge-sort on the list (Timsort[1]) ...

Reply:
When you remove the code snippets, the first one has 214 and the 2nd 259.  I thought the code snippets don't count towards the total word count of an insight? I made the requested change, just wondering about the wordcount